js-webapp-async/README.md

```diff
@@ -58,20 +60,15 @@ The webpage that uses the worker will look like:
 The worker code will contain only handling the incoming message. In this case, the worker will unpack the message, do the calculation, and pack the results in a new message that it will send back.

```

Comment:
```suggestion
The web worker code will contain only handling the incoming message. In this case, the web worker will unpack the message, do the calculation, and pack the results in a new message that it will send back.

```